shiny/http_server.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+
+
+app = Flask(__name__)
+
+
+@app.route('/v1/do-nothing/<unicorn_name>', methods=['POST'])
+def do_nothing(unicorn_name):
+    return ''
```

Comment:
This is getting way too complicated. Let's just table it until endpoints can do the translation for us.